docs/submitter.rst

```diff
@@ -5,7 +5,7 @@ Metadata Submitter Backend
 
 .. note:: Requirements:
 
-  - Python 3.8+
+  - Python 3.10+
```

Comment:
```suggestion
  - Python 3.11+
```
[.pylintrc](https://github.com/CSCfi/metadata-submitter/blob/60f5397748a2ac62cd707a0a1b56f177c2430eaf/.pylintrc#L83-L85) and [pyproject.toml](https://github.com/CSCfi/metadata-submitter/blob/60f5397748a2ac62cd707a0a1b56f177c2430eaf/pyproject.toml#L10) have minimum version set as `3.11`, should this match that?